test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestOpenJ9DiagnosticsMXBean.java

```diff
@@ -69,6 +71,7 @@
 	private JMXConnector connector = null;
 	private static ProcessLocking lock;
 	private static String tmpFileName;
+	private static boolean zOS = "z/OS".equalsIgnoreCase(os) ? true : false;
```

Comment:
The field should be `final`.

Why the use of the ternary operator? Why not just `"z/OS".equalsIgnoreCase(os)`?